felix/dataplane/linux/int_dataplane.go

```diff
@@ -709,9 +710,17 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		}
 	}
 
-	routeTableV4 := routetable.New(interfaceRegexes, 4, false, config.NetlinkTimeout,
-		config.DeviceRouteSourceAddress, config.DeviceRouteProtocol, config.RemoveExternalRoutes, 0,
-		dp.loopSummarizer)
+	var routeTableV4 routeTable
+	routeTableV4 = &routetable.DummyTable{}
```

Comment:
This only covers one instance of the RouteTable, there can be multiple:
* One for IPv4, one for IPv6
* One for VXLAN
* etc.